frontend/public/locales/en/translation.json

```diff
@@ -122,10 +122,11 @@
         "send": "Send"
       },
       "commentSection": {
-        "commentsSection": "Comments section",
+        "commentsSection": "Comments section,",
```

Comment:
In my opinion, it would be better to call this just 'Comments' without the word 'section'.  And in Ukrainian the same.

Reply:
if Comments section - after that present phrase comments length
if will be Comments, Comments length - looks like not good because - repeats